vespajlib/src/main/java/com/yahoo/text/Text.java

```diff
@@ -118,19 +118,31 @@ public static OptionalInt validateTextString(String string) {
      * Validates that the given string value only contains text characters.
      */
     public static boolean isValidTextString(String string) {
-        for (int i = 0; i < string.length(); ) {
+        int length = string.length();
```

Comment:
This doesn't gain anything?

Reply:
No, not in benchmark. Change is intentional to see if it effects inconsistent JIT compilation, which sometimes leaves
all the String.functions here non inlined. Here, even non-optimized we should see it being called fewer times than codePointAt().